tests/Http/Three/SignupTest.php

```diff
@@ -616,7 +675,7 @@ public function testValidationForUpdatingASignup()
     {
         $signup = factory(Signup::class)->create();
 
-        $response = $this->withAdminAccessToken()->patchJson('api/v3/signups/' . $signup->id);
```

Comment:
Not required to merge, but you should be able to change all these back to `->withAdminAccessToken()` now.

Reply:
oh right ! I will add this in the card when I dig into the `AsOwner` tests to also change these back!